src/App.tsx

```diff
@@ -66,6 +68,24 @@ const App: React.FC = () => {
     }
   }, [setAppInfo, setUserInfo, client]);
 
+  const executeWfsDescribeFeatureType = useExecuteWfsDescribeFeatureType();
+  const getPropertyNames = useCallback(async (layerId: number | undefined) => {
+    let response: DescribeFeatureType | undefined;
+    const propertyNames: string[] = [];
+    if (layerSuggestionList && layerId) {
+      const layer = layerSuggestionList.filter(item => item.id === layerId)[0];
```

Comment:
The state `layerSuggestionList` is actually filled within the `registerLayerIdCompletionProvider` method. Can we refactor this logic to an own method?

Reply:
resolved in new commit